generate-docs/script-inputs/office_preview.d.ts

```diff
@@ -7886,6 +7886,46 @@ declare namespace Office {
 
 declare namespace Office {
     namespace MailboxEnums {
+        /**
+         * Specifies the sensitivity type of an appointment.
+         *
+         * [Api set: Mailbox Preview]
+         *
+         * @remarks
+         *
+         * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose or Read
+         * 
+         * @beta
+         */
+        enum AppointmentSensitivityType {
+            /**
+             * Normal.
```

Comment:
Consider embellishing these descriptions, since this will read in the docs as "Normal" three times in a row. Consider using the descriptions found here: https://support.office.com/en-us/article/Mark-your-email-as-Normal-Personal-Private-or-Confidential-4A76D05B-6C29-4A0D-9096-71784A6B12C1

Reply:
@AlexJerabek Thanks for this feedback. I'll incorporate in a subsequent check-in.